packages/bitcoin/build.sh

```diff
@@ -0,0 +1,37 @@
+TERMUX_PKG_HOMEPAGE=https://bitcoincore.org/ 
+TERMUX_PKG_DESCRIPTION="Bitcoin Core" 
+TERMUX_PKG_LICENSE="MIT" 
+TERMUX_PKG_VERSION=0.18.1 
+TERMUX_PKG_SRCURL=https://github.com/bitcoin/bitcoin/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=db3c76ac913adfd67e3c7ff243b443c9706f81dd382d1212875fefc2de1ea5ff 
+TERMUX_PKG_DEPENDS="libevent,libzmq,boost,termux-services"
+TERMUX_PKG_BUILD_DEPENDS="pkg-config,util-linux"
+TERMUX_PKG_BUILD_IN_SRC=1
+TERMUX_PKG_BUILDDIR=$TERMUX_PKG_SRCDIR
+
+termux_step_patch_package() {
+    patch -p1 < $TERMUX_PKG_BUILDER_DIR/0001-android-patches.patch
```

Comment:
`termux_step_patch_package()` shouldn't be overridden. Default one is enough.

Reply:
I'm not so sure about this. I have copied the - admittedly complex - patch strategy from upstream (https://github.com/greenaddress/bitcoin_ndk) and had assumed there is a reason for it. Simply leaving the patch files in the package dir doesn't work: 
```
1 out of 1 hunk FAILED -- saving rejects to file configure.ac.rej
The text leading up to this was:
--------------------------
|diff --git a/evutil_rand.c b/evutil_rand.c
|index 046a14b..3f0bf2c 100644
|--- a/evutil_rand.c
|+++ b/evutil_rand.c
--------------------------
```

Reply:
If `patch -p1` doesn't work, it will not work here anyway.

Reply:
0001-fixup.patch can be applied only with `-p0` and therefore should be modified.